packages/devtools-kit/src/ctx/hook.ts

```diff
@@ -230,6 +230,9 @@ export interface DevToolsContextHooks extends DevToolsV6PluginAPIHooks {
   [DevToolsContextHookKeys.COMPONENT_UNHIGHLIGHT]: () => void
 }
 
+const MAX_$VM = 10
+const $vmQueue: any[] = []
```

Comment:
How about using WeakMap here?

As the component instance may be destroyed, and the reference relationship should be released.